Gemfile

```diff
@@ -49,6 +49,10 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'sentry-sidekiq'
 
+gem 'faraday'
+gem 'faraday_middleware'
```

Comment:
@Jullianos @AndriiMysko `faraday_middleware` is deprecated.
Why you are not using `rest-client` instead of `faraday`?

Reply:
I've decided to go with `faraday_middleware` because that was the gem used in travis-api from where I took the RemoteLog code.

Reply:
@Jullianos you can't decide by yourself. Every library used should be approved by our side: See 5.4 in PRD.
Gem `faraday_middleware` is deprecated. We are not going to use unsupported code in new applications. Pls use `rest-client`

Reply:
I've refactored the code and removed faraday_middleware and faraday as dependencies.